engine/swarm/ingress.md

```diff
@@ -119,14 +119,14 @@ $ docker service create --name dns-cache \
   dns-cache
 ```
 
-#### TCP and UDP
+#### TCP only
 
 **Long syntax:**
 
 ```bash
 $ docker service create --name dns-cache \
   --publish published=53,target=53 \
```

Comment:
The heading TCP and UDP is correct for this section as L128 is for TCP (which is the default) and 129 is to set UDP